platformio.ini

```diff
@@ -167,53 +169,53 @@ build_flags =
   -D RX_ELRS_GENERIC_2400_TD_PA_ESP32
 
 
-[env:rx-iflight-elrs-td-2400]
-extends = env:rx-generic-2400-td-pa
+;-- generic 2.4 GHz ELRS boards with overlays or other variations
+
+[env:rx-generic-2400-pa-dcdc]           ; like generic 2400 pa, but sx uses DCDC mode
```

Comment:
May make since to remove as no hardware exists for this target.

Reply:
DONE. Thx